.pyup.yml

```diff
@@ -3,5 +3,4 @@ update: all
 schedule: every week
 label_prs: update
 assignees:
-  - wlonk
-  - jgerigmeyer
+  - davisagli
```

Comment:
So we're getting dependency updates here, and then @davisagli will regularly be pulling from here into the authoritative repo, yes?

Reply:
Not quite. For MetaShare we're getting updates on the OddBird fork, then @davisagli can merge those upstream along with other changes. For MetaDeploy, since we're not doing active development, I left PyUp/Greenkeeper configured to use the SFDO-Tooling repo. So David will merge those if he wants. (If there's a reason we want to do it, we can always pull the changes over to our repo and then submit a PR upstream.)

Reply:
OK, sounds good.